packages/shorebird_code_push_protocol/lib/src/models/user.dart

```diff
@@ -28,4 +29,9 @@ class User {
 
   /// Whether the user is currently a paying customer.
   final bool hasActiveSubscription;
+
+  /// The user's Stripe customer id.
+  ///
+  /// This will be null for users that have never paid.
+  final String? stripeCustomerId;
```

Comment:
Why is this necessary? I don't think we should expose this publicly imo.

Reply:
We need this on the server side, but I agree about not exposing it to the client – maybe we should create a different User model for the server?

Reply:
Yeah I think we should create a UserEntity on the DB side which strictly corresponds to the DB schema and keep that decoupled from the public models.